src/assets/js/full-post-page/unlock-content/edit-full-post-page.js

```diff
@@ -0,0 +1,282 @@
+import { __ } from "@wordpress/i18n";
+import {
+  PanelBody,
+  PanelRow,
+  TextControl,
+  SelectControl,
+  Button,
+  Notice,
+} from "@wordpress/components";
+import { useEffect, useState } from "react";
+import apiFetch from "@wordpress/api-fetch";
+import { useSelect } from '@wordpress/data';
+import "../../../scss/admin/editor.scss";
+
+const EditFullPostPage = ({ locks, ethereumNetworks, onUpdateLocks, onUpdateEthereumNetworks }) => {
+
+  const [saveMessage, setSaveMessage] = useState(null);
+  const [refresh, setRefresh] = useState(false); //refresh lock(s) meta panel to keep track of locks already added
+
+
+  //get current post id
+  const post = useSelect((select) => {
```

Comment:
If you are getting the post's id here it does not make sense to have the post's locks and ethereumNetworks as props...